server/RestaurantApp.BLL/Services/JWTService/JWTService.cs

```diff
@@ -57,7 +57,7 @@ public Account GetCurrentAccount()
                     var jwtToken = (JwtSecurityToken)validatedToken;
                     var accountId = int.Parse(jwtToken.Claims.First(x => x.Type == "id").Value);
 
-                    return _unitOfWork.Accounts.Get(accountId);
+                    return _unitOfWork.Accounts.GetAccountWithCard(accountId);
```

Comment:
Maybe pass in a Boolean to check whether include card or not. Sometime we don’t want to include the card